codalab/apps/api/views/worksheet_views.py

```diff
@@ -298,9 +298,9 @@ def post(self, request, uuid):
 
                 # Convert to ints
                 for key in ['request_cpus', 'request_gpus', 'request_priority']:
```

Comment:
Given that you're now getting the metadata specs, you shouldn't need to hard code any of the metadata keys here - we should just be able to get the type.